bitnami/node/templates/deployment.yaml

```diff
@@ -23,6 +23,8 @@ spec:
         {{- include "common.tplvalues.render" (dict "value" .Values.podLabels "context" $) | nindent 8 }}
         {{- end }}
     spec:
+      automountServiceAccountToken: {{ .Values.serviceAccount.autoMount }}
```

Comment:
Please add `automountServiceAccountToken` in the serviceAccount.yaml, not at pod level.

Adding granular `automountServiceAccountToken` at pod-level would be a different feature we do not support yet.